spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -357,7 +359,9 @@ The value of the counter must be equivalent to a call to the instance Counter's
 {
   "simple_responseTime": {
     "count": 1,
-    "elapsedTime": 12300000000
+    "elapsedTime": 12300000000,
+    "maxTimeDuration": 3231000000,
+    "minTimeDuration": 25600000
```

Comment:
There should be a note somewhere around here that it will be null if no measurements are available

Reply:
note added